app/services/search/query_builders/chat_channel_membership.rb

```diff
@@ -0,0 +1,77 @@
+module Search
+  module QueryBuilders
+    class ChatChannelMembership
+      FILTER_KEYS = %i[
+        channel_status
+        channel_type
+        status
+        viewable_by
+      ].freeze
+
+      QUERY_KEYS = %i[
+        channel_text
+      ].freeze
+
+      DEFAULT_PARAMS = {
+        sort_by: "channel_last_message_at",
+        sort_direction: "desc",
+        size: 0
```

Comment:
what's size?

Reply:
How many documents are returned. Elasticsearch by default returns 10, I wanted to make our default size 0 to make sure people are explicit about requesting the correct number.

Reply:
Got it! +1 for adding this to the inline documentation of the search method :D As, according to:

https://github.com/thepracticaldev/dev.to/pull/6217/files/573a6f6044d0a76ea8b19ad504937c8230c97df4#diff-c8d5692235cc1a81cb7ed64d40688e2cR53

it won't break, just do a HTTP call that will always return 0 results